pom.xml

```diff
@@ -167,15 +168,15 @@
                 <groupId>org.liquibase</groupId>
                 <artifactId>liquibase-maven-plugin</artifactId>
                 <configuration>
-                  <changeLogFile>
-                    ${basedir}/src/main/resources/db/db-incremental.xml
-                  </changeLogFile>
-                  <driver>com.mysql.cj.jdbc.Driver</driver>
-                  <url>jdbc:mysql://localhost:3306/efeeder?nullNamePatternMatchesAll=true</url>
-                  <username>root</username>
-                  <password>root</password>
+                    <changeLogFile>
+                        ${basedir}/src/main/resources/db/db-incremental.xml
+                    </changeLogFile>
+                    <driver>com.mysql.cj.jdbc.Driver</driver>
+                    <url>jdbc:mysql://localhost:3306/efeeder?nullNamePatternMatchesAll=true</url>
+                    <username>root</username>
+                    <password>root</password>
                 </configuration>
```

Comment:
I think we haven't define how many spaces we should use for identation in html files. it would be good to define them @ziur @Ramir0 @rayaviri.  As Amir commented to me 2 spaces could be good probably?